src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
+    public function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
```

Comment:
How are set those options ? I mean, the method is never called with the options parameter ?